### DISCUSSION TITLE
Toast intent error icon color does not match Figma design guide

### DISCUSSION THREAD

**Discussion by cherrycoffeecake** [microsoft/fluentui#30604] [Q&A] [answered]
Is this a bug in the React library? Based on the color token names alone it seems like a bug? 

![image](https://github.com/microsoft/fluentui/assets/128184361/d1aad50d-f1c1-4632-97fb-48878f73ed82)

**React v9.46.4**  [link](https://react.fluentui.dev/?path=/docs/components-toast--default)
Toast “intent: error”
red = `#6E0811`; aka `var(--colorPaletteCranberryForeground2)`

**Figma Microsoft Fluent 2 Web (08.17.2023)** [link](https://www.figma.com/community/file/836828295772957889/microsoft-fluent-2-web)
“Toast” component with “Status state: ErrorCircle”
red = `#D13438`; aka `StatusDangerForeground3`

**Comment by cherrycoffeecake** [comment]
Is this the correct place to ask this question? If not, where should I direct my concern?

**Comment by tombohub** [comment]
Could you please post a link to that component so people can click?

**Comment by cherrycoffeecake** [comment]
Sure thing! Updated with links to the React page and the Figma Microsoft Fluent 2 Web file.

**Comment by behowell** [comment]
It looks like @ling1726 had issue https://github.com/microsoft/fluentui/issues/28219 to update the colors of the toast icons, but it's closed now. I'm not sure if it was closed intentionally or not, since the code still has [FIXME comments](https://github.com/microsoft/fluentui/blob/938958cbfd9058eb7681c8feddfdd4e0538733be/packages/react-components/react-toast/src/components/ToastTitle/useToastTitleStyles.styles.ts#L56). I'll follow up to see what the status of that issue is.

**Reply by ling1726** [reply, marked as answer]
When the Toast was implemented those semantic color tokens weren't available. I've updated the colours in https://github.com/microsoft/fluentui/pull/30715.

However those colours still don't match what's linked in Fluent 2 here. We'll need to follow up with the designers
